frontend/projects/ui/src/app/pages/server-routes/server-specs/server-specs.page.ts

```diff
@@ -2,6 +2,7 @@ import { ChangeDetectionStrategy, Component } from '@angular/core'
 import { ToastController } from '@ionic/angular'
```

Comment:
```suggestion
import { ModalController, ToastController } from '@ionic/angular'
```

Reply:
I cannot comment on the direct line, but you also need to add the following to the class constructor:
```
private readonly modalCtrl: ModalController,
```

so its finalized form should look like:
```
  constructor(
    private readonly toastCtrl: ToastController,
    private readonly modalCtrl: ModalController,
    private readonly patch: PatchDB<DataModel>,
    private readonly config: ConfigService,
  ) {}
```

Reply:
Hey @elvece thank you for testing it so quickly! :rocket: Committed the fix.